sql/moz-fx-data-shared-prod/firefox_desktop_derived/pageload_1pct_v1/metadata.yaml

```
@@ -0,0 +1,23 @@
+friendly_name: Pageload 1pct
+description: |-
+  Contains 1pct sample from pageload_v1.  Document_id was used to sample data since most pings did not have client_id.
+owners:
+- wichan@mozilla.com
+- acreskey@mozilla.com
+- dpalmeiro@mozilla.com
+labels:
+  incremental: true
+scheduling:
+  dag_name: bqetl_pageload_v1
+  task_name: firefox_desktop_pageload_1pct__v1
```

You might need to set `--schema_update_option=ALLOW_FIELD_ADDITION'` since the schema is based on the ping instead of schema.yaml
```suggestion
scheduling:
  dag_name: bqetl_pageload_v1
  task_name: firefox_desktop_pageload_1pct__v1
  arguments: ['--schema_update_option=ALLOW_FIELD_ADDITION']
```